lava/chain.json

```diff
@@ -0,0 +1,209 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "lava-testnet-1",
+    "status": "live",
+    "network_type": "testnet",
```

Comment:
since this is a testnet, can we please move all of this into the /testnets/ directory?

Reply:
Corrected:
[cfa6968](https://github.com/cosmos/chain-registry/pull/2244/commits/cfa6968697836fa1bd0e2d5cf47750789ca1284a)